verbose: true

scope_timer:
  enable: false
  log_filename: "" # used, if not empty

initialize_from_odom: false

mapping:
  remap_tf: false
  rate: 10.0 # [Hz]
  publish_rate: 0.5 # [Hz]
  line_resolution: 0.2
  plane_resolution: 0.4
